src/store/cart/__test__/reducer.test.js

```diff
@@ -1,41 +1,98 @@
-import reducer from '../reducer';
-import * as types from '../types';
+import reducer, { initialState } from '../reducer';
+import * as action from '../actionCreators';
 
 describe('cart reducer', () => {
-  const product = [
-    {
-      id: '1',
-      name: 'fx',
-      price: 400,
-      pcs: 2,
-      totalPrice: 800,
+  const product = {
+    id: '1',
+    productId: '2',
+    name: 'fx',
+    price: 400,
+    pcs: 2,
+    totalPrice: 800,
+  };
+
+  const updateProduct = {
+    id: '1',
+    productId: '2',
+    name: 'fx',
+    price: 400,
+    pcs: 4,
+    totalPrice: 1600,
+  };
+
+  const error = 'error';
+
+  const stateAfterAddProduct = {
```

Comment:
Specific data can be put to their tests or describe blocks wherever they are used, so they do not pollute global setup space